src/main/java/com/redhat/cloud/policies/app/rest/PolicyCrudService.java

```diff
@@ -584,7 +584,6 @@ public Response updatePolicy(@QueryParam("dry") boolean dryRun, @PathParam("poli
                 // All is good, we can now do the real work
                 try {
                     storedPolicy.populateFrom(policy);
-                    storedPolicy.customerid = user.getAccount();
```

Comment:
`storedPolicy` comes from the DB and already has a `customerid` value. It doesn't make sense to change it here.